packages/launcher/test/unit/windows_spec.ts

```diff
@@ -35,6 +35,13 @@ function detect (goalBrowsers: Browser[]) {
 
 const HOMEDIR = 'C:/Users/flotwig'
 
+// Electron and WebKit can be safely removed since Electron is baked into the Cypress binary
```

Comment:
I was wrong about Webkit - I'm very sorry! Thank you @MikeMcC399 for bringing this up - I was going off of what is generally available on Windows, and did not realize we were recommending the use of playwright-webkit for windows support.